configuration_files/trajectory_builder_3d.lua

```diff
@@ -64,10 +64,27 @@ TRAJECTORY_BUILDER_3D = {
   -- TODO(schwoere,wohe): Remove this constant. This is only kept for ROS.
   imu_gravity_time_constant = 10.,
   pose_extrapolator = {
+    use_imu_based = false,
     constant_velocity = {
       imu_gravity_time_constant = 10.,
       pose_queue_duration = 0.001,
     },
+    -- TODO(wohe): Tune these parameters on the example datasets.
```

Comment:
@wohe I added a TODO for the final tuning and end-to-end testing on the official datasets. I hope that's okay.

Reply:
SGTM